.travis.yml

```diff
@@ -64,10 +63,13 @@ script:
         *)                   DOCKER=$TARGET ;;
       esac;
       if [ -n "$DOCKER" ]; then
-          sh ci/build-run-docker.sh "$DOCKER" "$TARGET" "$SKIP_TESTS";
```

Comment:
Since the scripts are POSIX-compatible, it could instead use `sh` instead of `bash`.

Reply:
Done. Let's see if travis is green.